README.md

```diff
@@ -224,6 +224,37 @@ $ cargo raze
 You can now depend on any _explicit_ dependencies in any Rust rule by depending on
 `//cargo:your_dependency_name`.
 
+### Using cargo-raze through Bazel
+
+Cargo-raze can be built entirely in Bazel and used without needing to setup cargo
+on the host machine. To do so, simply add the following to the WORKSPACE file in
+your project:
+
+```python
+# TODO; Update, 0.10.0 does not contain these changes!
+http_archive(
+    name = "cargo_raze",
+    # sha256 = "accb5a89cbe63d55dcdae85938e56ff3aa56f21eb847ed826a28a83db8500ae6",
+    strip_prefix = "cargo-raze-0.10.0",
+    url = "https://github.com/google/cargo-raze/archive/v0.10.0.tar.gz",
+)
+
+load("@cargo_raze//:repositories.bzl", "cargo_raze_repositories")
```

Comment:
Should this include transitive_deps?

(This seems to provide little or no convenience otherwise over just making users include @cargo_raze//third_party:third_party_repositories and @cargo_raze//third_party:third_party_transitive_deps)

Reply:
It's more to solidify the user facing API. I think this is more consistent with other projects (namely, buildifier) so wanted to adopt this API. I'd honestly be in favor of dropping the `third_party` `*.bzl` files in favor of this. Your thoughts?

Reply:
I don't feel strongly about this. IIUC your suggestion around dropping third_party bzl, it sgtm.

Reply:
Can do, give me a sec

Reply:
Done